memtagabielf64/TRADEMARK_NOTICE

```diff
@@ -1,6 +1,3 @@
-The text of and illustrations in this document are licensed by Arm
```

Comment:
I agree that the "by Arm" words here aren't fully correct now that other entities are also contributing to these specifications.
I'll address that issue for the TRADEMARK_NOTICE files in the other ABI docs in a separate pull request that I will put up shortly.
Here, please only remove the 2 words "by Arm" at the end of the first line. The separate pull request I'll put up shortly will make the same change to the TRADEMARK_NOTICE files in the other ABI docs.

Reply:
I've now created pull request #170 to address this issue for the TRADEMARK_NOTICE files in the other ABI docs.